ledgereth/__main__.py

```diff
@@ -86,9 +86,24 @@ def get_args(argv):
         "-p",
         "--gasprice",
         type=int,
-        required=True,
+        required=False,
```

Comment:
`required=False` might already be the default and you could exclude these if you wanted